content-repo/gendocs.py

```diff
@@ -516,25 +498,21 @@ def process_extra_docs(target_dir: str, prefix: str,
 
 
 def process_doc_info(doc_info: DocInfo, success: List[str], fail: List[str], doc_infos: List[DocInfo],
-                     seen_docs: Dict[str, DocInfo], private_doc: bool = False):
+                     seen_docs: Dict[str, DocInfo]):
     if doc_info.error_msg == EMPTY_FILE_MSG or IGNORE_MSG in (doc_info.error_msg or ''):
         # skip empty and ignored files.
         return
     if doc_info.error_msg:
         fail.append(f'{doc_info.readme} ({doc_info.error_msg})')
     elif doc_info.id in seen_docs:
-        if private_doc or not version_conflict(doc_info, seen_docs[doc_info.id]):
```

Comment:
Please leave the second condition
```suggestion
        if not version_conflict(doc_info, seen_docs[doc_info.id]):
```